docs/packages/xstate-vue/index.md

```diff
@@ -104,12 +107,13 @@ _Since 0.5.0_
   - Defaults to attempting to get the `actor.state`, or returning `undefined` if that does not exist.
 
 ```js
+import { computed } from "vue";
 import { useActor } from '@xstate/vue';
 
 export default {
   props: ['someSpawnedActor'],
   setup(props) {
-    const { state, send } = useActor(props.someSpawnedActor);
+    const { state, send } = useActor(computed(() => props.someSpawnedActor));
```

Comment:
Since this is the main example here - this might still look like a requirement. I would just push the note about it + a single-line example to the end of this section (so basically below this example here).

Reply:
ahhh I see. 👍